src/qibo/models/circuit.py

```diff
@@ -120,9 +120,12 @@ class Circuit:
     A specific backend has to be used for performing calculations.
 
     Circuits can be created with a specific number of qubits and wire names.
+
     Example:
-        .. testcode::
+        .. code-block:: python
```

Comment:
Just to understand: was there any issue with the `testcode` directive?

Reply:
`testcode` does not display the code in the compiled documentation, leaving the example blank in the doc https://qibo.science/qibo/stable/api-reference/qibo.html#qibo.models.circuit.Circuit

Reply:
Ok. I'm not sure why it is happening, since in principle even `testcode` blocks should be shown, according to Sphinx
https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html#directive-testcode
> By default it is shown as a highlighted code block.

(there is an explicit attribute `:hide:` to avoid showing it).

I wonder why... but a workaround is still better than an empty block. Thanks for the fix!